File: c#/program21.cs
using System;

class Pattern
{
    public int iRow;
    public int iCol;

    public Pattern(int no1, int no2)
    {
        this.iRow = no1;
        this.iCol = no2;
    }

    public void DisplayPattern()
    {
        int i = 0, j = 0;

        for(i = this.iRow; i >= 1; i--)
        {
            for(j = this.iCol; j >= 1; j--)
            {
                Console.Write(i+"\t");
            }
            Console.WriteLine("\n");
        }
    }
}

class program21
{
    static void Main(string[] args)
    {
        int value1 = 0, value2 = 0;

        Console.WriteLine("Enter the number of rows :");
        value1 = int.Parse(Console.ReadLine());

        Console.WriteLine("Enter the number of columns :");
        value2 = int.Parse(Console.ReadLine());

        Pattern obj = new Pattern(value1,value2);
        obj.DisplayPattern();
    }
}

File: java/program21.java
import java.util.*;

class Pattern
{
    public int iRow;
    public int iCol;

    public Pattern(int X,int Y)
    {
        iRow = X;
        iCol = Y;
    }

    public void DisplayPattern()
    {
        int i = 0,j = 0;

        for(i = 1; i <= iRow; i++)
        {
            for(j = 1; j <= iCol; j++)
            {
                System.out.print(i+"\t");
            }
            System.out.println();
        }
    }
}

class program21
{
    public static void main(String arg[])
    {
        int iValue = 0,iValue1 = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number of rows ");
        iValue = sobj.nextInt();

        System.out.println("Enter number of columns ");
        iValue1 = sobj.nextInt();

        Pattern obj = new Pattern(iValue,iValue1);
        obj.DisplayPattern();
    }
}